staging/src/k8s.io/cri-api/SECURITY_CONTACTS

```diff
@@ -0,0 +1,13 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+tallclair
```

Comment:
@tallclair, I am not sure what the responsibility of the security contact is. A couple of questions:
 1. Are you okay with being the contact?
 2. Should there be multiple contacts for fail-over?

Reply:
Yes to both. The security contact should be someone we can reach out to for help with resolving a security issue in the repository. If this is just holding the API, it's unlikely to matter much (but still required). Any trusted owner can be a security contact.

Reply:
Ack. 

@dims could you add me to the list of security contacts?

Reply:
Done!